src/container/containerManager.ts

```diff
@@ -110,20 +116,27 @@ export class ContainerManager {
     }
 
     private async generateDeploymentString(templateFile: string,
-                                           deployFile: string,
+                                           configPath: string,
```

Comment:
The method name `generateDeploymentString` is no longer applicable now as the return value is not a string.